### ISSUE TITLE
Null ref: T2.DungeonRoomTransitioner.Update

### ISSUE THREAD

**Issue by ChrisNZL** [ChrisNZL/Tallowmere2#541]
Auto bug report from 0.1.3b.

Feedback ID: 20200515-STHQL

```
6:10:30, Frame 22003, EXCEPTION »  NullReferenceException: Object reference not set to an instance of an object
>>>>> CRITICAL ERROR >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>

T2.DungeonRoomTransitioner.Update ()

GameStates: RompingThroughDungeon, GameOverScreen
GameSetupMode: Singleplayer
Players: 1
Room: 0 / 6
RoomModifiers: None
```

**Comment by ChrisNZL** [comment]
Addressed in 0.1.3c.

Improved null checks and logging.

**Comment by ChrisNZL** [comment]
Still happening in 0.1.4.

Feedback ID: 20200606-JNNQD

```
5:57:31, Frame 84326, EXCEPTION »  NullReferenceException: Object reference not set to an instance of an object
>>>>> CRITICAL ERROR >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>

T2.DungeonRoomTransitioner.Update ()

GameStates: RompingThroughDungeon, GameOverScreen
GameSetupMode: Singleplayer
Players: 1
DungeonRoomTransitionerState: RevealingRoom
Room: 0 / 8
RoomModifiers: None
YouHaveDiedState: InitialDelay
```

**Comment by ChrisNZL** [comment]
Related: issue #545

**Comment by ChrisNZL** [comment]
Fixed in 0.1.5. Added further null checks.
